test/test_helper.rb

```diff
@@ -12,12 +12,12 @@
 
 Seedbank.application_root = Pathname.new(File.expand_path('../dummy', __FILE__))
 
-class Seedbank::Spec < MiniTest::Spec
+class Seedbank::Spec < Minitest::Spec
```

Comment:
As we've had to update the rails to more than 5 to work with (at least 5.2 ) the Minitest will get updated to 5.1 as a dependency of active support  and then we have to apply the renaming [fix](https://github.com/minitest/minitest/commit/8546e453aa28f427c29e2f750118ad1abe4e331d)